frontend/src/uxbox/config.cljs

```diff
@@ -18,6 +18,7 @@
 (let [config (gobj/get goog.global "uxboxConfig")
       public-url (gobj/get config "publicURL" "http://localhost:6060")]
 
-  (def default-language "en")
+  (def default-lang "en")
```

Comment:
Just a detail I think, but why rename this field ?

Reply:
Yeah, I think your are right. I will revert to `default-language`.